app/cdap/components/common/DataQualityCircularProgressBar/utils.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import {
+  IHeaderNamesList,
+  IDataQualityItem,
+} from 'components/WranglerGrid/SelectColumnPanel/types';
+import { IStatistics } from 'components/GridTable/types';
+
+export const getDataQuality = (statistics: IStatistics, columnList: IHeaderNamesList[]) => {
+  const dataQuality: IDataQualityItem[] = [];
+  columnList?.forEach((columnName: IHeaderNamesList) => {
+    const generalValues: IDataQualityItem = statistics[columnName.name].general;
+    if (generalValues.null) {
```

Comment:
What is this? generalValues.null? are you adding an attribute called null to this? Please do not use null as an attribute name. If not... what is this doing?

Reply:
null is the attribute we are getting from API, it contains the percentage of the null values with respect to column